src/views/List.jsx

```diff
@@ -12,35 +13,50 @@ export function List({ data }) {
 		item.name.toLowerCase().includes(searchItem.toLowerCase()),
 	);
 
+	const listName = listPath.slice(29);
+
 	return (
 		<>
-			<form onSubmit={(event) => event.preventDefault()}>
-				<label htmlFor="search-item">Search Item: </label>
-
-				<input
-					id="search-item"
-					type="search"
-					placeholder="Search Item..."
-					onChange={handleTextChange}
-					value={searchItem}
-				/>
-
-				{searchItem && (
-					<button
-						type="button"
-						onClick={() => {
-							setSearchItem('');
-						}}
-					>
-						X
-					</button>
-				)}
-			</form>
-			<ul>
-				{filteredItems.map((item) => {
-					return <ListItem key={item.id} name={item.name} />;
-				})}
-			</ul>
+			{data.length === 0 ? (
```

Comment:
Love the use of the ternary operator for conditional rendering here! A quick suggestion: consider using `!data.length` to make the condition even more concise!

Reply:
i appreciate it!